### PR TITLE
Ansh Tripathi's card

### PR DESCRIPTION
**Pull request by Anshtripathi079**
Added my card information

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -106,6 +106,33 @@ <h2>A project for first-time contributions</h2>
           </div>
         </div>
         <!-- ________*END TEMPLATE* Contributor card END ________  -->
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (TenTraicion) [inline index.html:110]:
```suggestion
        <!-- ________ *Ansh* Contributor card START ________  -->
```

File: index.html [hunk 2]
```diff
@@ -106,6 +106,33 @@ <h2>A project for first-time contributions</h2>
...
+
+        <div class="card">
+          <p class="name">Ansh Tripathi</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/" target="_blank">Anshtripathi079</a>
```
Comment (TenTraicion) [inline index.html:116]:
```suggestion
            <a href="https://www.github.com/Anshtripathi079" target="_blank">@Anshtripathi079</a>
```

File: index.html [hunk 3]
```diff
@@ -106,6 +106,33 @@ <h2>A project for first-time contributions</h2>
...
+                <a href="https://www.100daysofcode.com/" target="_blank" title="Second Resource">100daysofcode</a>
+              </li>
+              <li>
+                <a href="https://www.freecodecamp.org/" target="_blank" title="Third Resourse">Free code camp</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+
+         <!-- ________*END TEMPLATE* Contributor card END ________  -->
```
Comment (TenTraicion) [inline index.html:135]:
```suggestion
         <!-- ________*Ansh* Contributor card END ________  -->
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
Kindly do the recommend changes and move your card after line 117 where it says `Paste your card directly below this line`. Thanks

**Comment by CandidDeer** [comment]
Hi Ansh, thank you for making the requested changes. Your contribution has been merged and will be live shortly.

**Pull request merged**